ts/pages/account/dashboard.tsx

```diff
@@ -458,7 +458,7 @@ export const Account: React.FC<AccountProps> = () => {
                     ) : (
                         delegatorData.forCurrentEpoch.poolData
                             // Don't show pools with pending withdrawals, they are shown in pending section instead
-                            .filter(p => !pendingUnstakePoolSet.has(p.poolId))
+                            .filter(p => !pendingUnstakePoolSet.has(p.poolId) && p.zrxStaked > 0)
```

Comment:
there was an unrelated bug with the account page showing staking pools that you are currently staking 0 ZRX but are in the api response